pkgs/applications/misc/deco/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, scsh, feh, xlibs }:
```

Comment:
Changing `xlibs` to `xdpyinfo` in deco/default.nix results to: `error: undefined variable ‘xlibs’ at /pub/nixpkgs/pkgs/top-level/all-packages.nix:2044:59`